develop/CICD/Skyline Communications/Publish to Catalog/Publish_To_Catalog.md

```diff
@@ -2,11 +2,11 @@
 uid: publish_to_catalog
 ---
 
-# Publishing artifacts to the DataMiner Catalog
+# Publishing items on the DataMiner Catalog
```

Comment:
"on" seems like a weird choice of preposition here. I think "in" would sound more natural:
```suggestion
# Publishing items in the DataMiner Catalog
```

Note also that the corresponding title in the [toc.yml](https://github.com/SkylineCommunications/dataminer-docs/blob/main/develop/toc.yml) should also be adjusted if we adjust the title here.

Reply:
I've doubted about it, even asked ChatGPT. It suggested using "in" a catalog if it was an actual paper catalog, but "on" a catalog if it was an online platform.
But I'm OK with either!